pkg/util/hash/hash.go

```diff
@@ -35,8 +35,8 @@ func Hash(obj interface{}) (string, error) {
 	if err != nil {
 		return "", errors.Wrapf(err, "failed to convert the object to json encoded format")
 	}
-	hashBytes := md5.Sum(jsonEncodedValues)
-	return hex.EncodeToString(hashBytes[:]), nil
+	hashBytes := xxhash.Sum64(jsonEncodedValues)
+	return strconv.FormatUint(hashBytes, 10), nil
```

Comment:
Magic number `10` over here is something like in which format output string is required. 
For example: 
- number 2 states give a string for a given hash in form of binary.
  example output: `100000110001010110110000101101011001101101111101101010101111100`
- number 8 states give a string for a given hash in form of octal
  example output: `1462530262414137617526`
- number 10 states give a string for a given hash in form of decimal
   example output: `14747893672817139542`
- number 16 states give a string for a given hash in form of hexadecimal
   example output: `ccab0b28617f1f56`

Reply:
Yes, the second parameter is the base in which we want the output. I kept it 10 so that we can get in form of decimal. Also the xxhash algorithm output is in decimal format.